visualization/app/puppeteer.helper.ts

```diff
@@ -1,8 +1,12 @@
 import path from "path"
+import { setDefaultOptions } from 'expect-puppeteer'
 import { ConsoleMessage } from "puppeteer"
 
 export const CC_URL = `file:${path.join(__dirname, "../dist/webpack/index.html")}`
-
+// expect-puppeteer toClick timeout does not work it might be the reason of flaky tests,
+// changed default options globally
+// [toClick issue](https://github.com/smooth-code/jest-puppeteer/issues/202)
+setDefaultOptions({ timeout: 6000 }) 
```

Comment:
I have always a bad feeling with huge default timeouts 😟 But if it is necessary this shouldn't be a blocker